### PR TITLE
tidb-lightning: add contents about `REFERENCES`

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Frank945946 — APPROVED** [review]

File: tidb-lightning/tidb-lightning-requirements.md [hunk 1]
```diff
@@ -77,6 +77,10 @@ TiDB Lightning 导入数据时，根据导入方式和启用特性等，需要
    </tr>
 </table>
 
+> **注意：**
+>
+> 如果你没有为下游数据库用户分配 `REFERENCES` 权限，你也可以提前手动在下游数据库中创建好目标表，以确保顺利导入数据。
```
Comment (D3Hunter) [inline tidb-lightning/tidb-lightning-requirements.md:82]:
```suggestion
> 如果要导入的表没有使用外键，则不需要 `REFERENCES` 权限
>
> 如果你没有为下游数据库用户分配 `REFERENCES` 权限，你也可以提前手动在下游数据库中创建好目标表，以确保顺利导入数据。
```

File: tidb-lightning/tidb-lightning-requirements.md [hunk 2]
```diff
@@ -77,6 +77,11 @@ TiDB Lightning 导入数据时，根据导入方式和启用特性等，需要
    </tr>
 </table>
 
+> **注意：**
+>
+> - 如果目标表没有使用外键，则不需要 `REFERENCES` 权限。
+> - 如果你没有为下游数据库用户分配 `REFERENCES` 权限，你也可以提前手动在下游数据库中创建好目标表，以确保顺利导入数据。
```
Comment (lilin90) [inline tidb-lightning/tidb-lightning-requirements.md:83]:
```suggestion
> - 如果目标表使用了外键，你需要为下游数据库用户添加 `REFERENCES` 权限，或者提前手动在下游数据库中创建好目标表，以确保顺利导入数据。
```

File: tidb-lightning/tidb-lightning-requirements.md [hunk 3]
```diff
@@ -77,6 +77,11 @@ TiDB Lightning 导入数据时，根据导入方式和启用特性等，需要
    </tr>
 </table>
 
+> **注意：**
+>
+> - 如果目标表没有使用外键，则不需要 `REFERENCES` 权限。
+> - 如果目标表使用了外键，你需要为下游数据库用户添加 `REFERENCES` 权限，或者提前手动在下游数据库中创建好目标表，以确保顺利导入数据。
```
Comment (lilin90) [inline tidb-lightning/tidb-lightning-requirements.md:83]:
```suggestion
> - 如果目标表使用了外键，则需要为下游数据库用户添加 `REFERENCES` 权限，或者提前手动在下游数据库中创建好目标表，以确保顺利导入数据。
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**